data/json/effects.json

```diff
@@ -1803,6 +1824,7 @@
         "type": "effect_type",
         "id": "stim_overdose",
         "name": [ "Stimulant Overdose" ],
+        "desc": [ "You overdosed stimulants and your body is twitching." ],
```

Comment:
Recommend something like `You can't sit still as you feel your heart pounding out of your chest.  You probably overdosed on those stims.`